x-pack/test/functional_basic/apps/ml/permissions/group1/index.ts

```diff
@@ -34,7 +34,6 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
       await ml.testResources.resetKibanaTimeZone();
     });
 
-    loadTestFile(require.resolve('./permissions'));
-    loadTestFile(require.resolve('./data_visualizer'));
+    loadTestFile(require.resolve('./full_ml_access'));
```

Comment:
I knew this question might come as I had the same thought: when splitting is making things actually slower rather than fast :) 
Loading config, starting Kibana + ES, loading test data tests around 1m 30s in best scenario, so it might look like we are "loosing", but when you look at the actual [FTR config groups](https://buildkite.com/elastic/kibana-pull-request/builds/103740#0186029b-658f-401a-a69c-fe9ed50e1c7a) it is debatable:

```
  "ftr_configs_49": {
    "title": "FTR Configs #49",
    "expectedDurationMin": 36.8,
    "names": [
      "x-pack/test/functional/apps/saved_objects_management/config.ts",
      "test/plugin_functional/config.ts",
      "x-pack/test/saved_object_api_integration/spaces_only/config.ts",
      "x-pack/test/functional_basic/apps/ml/permissions/group1/config.ts",
      "x-pack/test/functional/apps/watcher/config.ts",
      "x-pack/test/localization/config.ja_jp.ts"
    ]
  },
```


```
"ftr_configs_4": {
    "title": "FTR Configs #5",
    "expectedDurationMin": 35.3,
    "names": [
      "test/functional/apps/visualize/group3/config.ts",
      "test/functional/apps/home/config.ts",
      "x-pack/test/functional_basic/apps/ml/permissions/group2/config.ts",
      "x-pack/test/cloud_security_posture_functional/config.ts"
    ]
  },
```

```
  "ftr_configs_6": {
    "title": "FTR Configs #7",
    "expectedDurationMin": 35.3,
    "names": [
      "test/functional/apps/visualize/group1/config.ts",
      "x-pack/test/functional/apps/ml/short_tests/config.ts",
      "x-pack/test/functional/apps/uptime/config.ts",
      "x-pack/test/functional/apps/advanced_settings/config.ts",
      "x-pack/test/functional_basic/apps/ml/permissions/group3/config.ts"
    ]
  },
```

split allows better allocation, 2 groups will finish within 35.3 min, however putting it all together will make any of this groups slower at least by 2 min (yes, others will run faster but we need all of them to run pretty the same time)

Reply:
I can give it a shot and we can compare results

Reply:
Done, timings look good!